src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 1; i <= 3; i++) {
+            Ball ball = Lottery.getRandomBall();
+            System.out.println("Ball " + i + ": " + ball);
```

Comment:
In situations like this, there is no need to store the object in a local variable if it will be immediately used. Instead, you can pass/use the expression directly without assigning it to a variable